beetsplug/convert.py

```diff
@@ -336,14 +338,33 @@ def copy_album_art(self, album, dest_dir, path_formats, pretend=False):
                            util.displayable_path(album.artpath))
             return
 
-        if pretend:
-            self._log.info(u'cp {0} {1}',
-                           util.displayable_path(album.artpath),
-                           util.displayable_path(dest))
+        resize = False
+        maxwidth = None
+
+        if self.config['copy_album_art_maxwidth']:
+          maxwidth = self.config['copy_album_art_maxwidth'].get(int)
```

Comment:
Oops, sorry -- developing on a different machine to that running beets, so there's been a bit of back and forth.